app/_src/production/use-kuma.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Use Kuma
+---
+
+After {{site.mesh_product_name}} is installed, you can access the control plane via the following methods:
+
+| Access method | Mode | Permissions | 
```

Comment:
Should we be listing all the ways you can access kuma in the top section? It feels like we're presenting a lot of information that is not 100% needed to a novice user. To me if you first encounter Kuma you should be presented with 3 things:
1. `kumactl` for universal
2. `kubectl` for k8s
3. GUI for convenience 

And then we could have an advanced section with API server, and even using `kumactl` on k8s for read only ops.

Reply:
Side note: something for us to figure out, could we just have `kumactl` and translate the calls to `kubectl`? That would make it easier.

Reply:
> To me if you first encounter Kuma you should be presented with 3 things:

This page will be the second time they encounter Kuma. The plan is to create a Quickstart page that explains those three things you listed in addition to using kind, using kubernetes, do standalone, and apply traffic permission. The Quickstart will help people try Kuma and then the "Install kumactl" and "Use Kuma" pages would be for when they want to have a production instance of Kuma.